### PR TITLE
Fix ignite agregate metrics not being reported

### PR DESCRIPTION
**Pull request by emcastillo**
Fix the error introduced in #13 

The ignite metrics were not reported in the correct scope

### CODE REVIEW

**Comment by emcastillo** [comment]
I just added a test to verify that both, internally reported values are being correctly averaged, and that `ignite.metrics` are accessible at the reporter observation.

Thanks to @corochann for giving us the initial draft to test the ignite evaluators.

File: tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_evaluator.py [hunk 1]
```diff
@@ -188,3 +188,70 @@ def test_evaluator_progress_bar():
...
+        model()
+        # We return fake results for the reporters to
+        # and metrics to work
+        return (y_pred, y_pred)
+
+    evaluator = Engine(update_fn)
+    return evaluator
+
+
+class IgniteDataset(torch.utils.data.Dataset):
```
Comment (corochann) [inline tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_evaluator.py:219]: I noticed shorter way to write dataset is:

```
n_data = 10
x = torch.randn((n_data , 2), requires_grad=True)
y = torch.randn((n_data , 2))
dataset = TensorDataset(y, t)
```

Comment (emcastillo) [reply tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_evaluator.py]: Solved! thanks :D

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**